Segment-Mixpanel.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0.0'
+  s.dependency 'Analytics', '~> 3'
```

Comment:
@emkman @antonzy My 2 cents here: It should probably be `~> 3.2` because there is no way to know whether it will be compatible with analytics 3.3 and later (which is what `~> 3` implies). It drops support for dependencies that rely on analytics `3.0.0 -> 3.1.x` but in this case people can just choose to not upgrade.